### PR TITLE
Adds some quality of life to the Cafe

### PR DESCRIPTION
**Pull request by MidoriWroth**
## About The Pull Request

After the removal of the chemistry lab and autolathe from the Cafe, I decided to add a YouTool vendor, so people have at least some access to tools without stealing the ones from the Vox Box area. This will also add the four new imported food vendors to the Cafe lobby area, and the Syndicate clothing vendor to the hidden cave next to the Ash clothing vendor. Finally, this also adds a 30-sheet stack of plasma, uranium, gold, and silver in the bar backroom, for use in making exotic drinks.

## How This Contributes To The Skyrat Roleplay Experience

Having the YouTool added in allows easier access to tools, so people can create more custom things without taking the toolbelts in the Vox Box, and food vendors allow easier access to the newly added exotic foods. The Syndicate vendor allows the use of the Syndicate clothing, and as said above, the mineral stacks allow for better drink creations.

## Changelog

:cl:
qol: Added a YouTool and Syndicate clothing vendor to the Cafe
qol: Added the imported food vendors to the Cafe
qol: Gave the Cafe back some minerals for special drinks
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
There are already tools in the lockers, there are full belts of power tools in the same room the autolathe sat in. The YouTool is not really needed.

The mineral stacks should be trimmed down to 5 or so, I do not see such a high volume of drinks being made to warrant 600 units of each mineral chemical. Otherwise people will use them exclusively for building, which ~~could~~ will be abused with plasma and uranium. Mineral variety was removed previously so this will need well-justified. 
Aside from plasma, all the drink mixes use minerals in very small amounts, so it should not be an issue to trim down the amount.

Minerals for drinks should be in the bar directly, to make it clear they are not for building materials.

**Comment by MidoriWroth** [comment]
huh, had no idea those belts were actually in those lockers. even so, it would still be helpful to have the youtool there just in case =)

fair enough with the mineral amounts. people who abuse them could just be punished for doing so but whatever.

File: _maps/map_files/generic/CentCom_skyrat_z2.dmm [hunk 1]
```diff
@@ -6436,7 +6437,12 @@
 	dir = 8
 	},
 /obj/effect/turf_decal/trimline/blue/filled/line,
-/obj/machinery/vending/autodrobe,
+/obj/machinery/vending/imported{
```
Comment (YakumoChen) [inline _maps/map_files/generic/CentCom_skyrat_z2.dmm:6439]: you don't need to manually set the prices on these. All vending machines on other z-levels have their prices zeroed.

File: _maps/map_files/generic/CentCom_skyrat_z2.dmm [hunk 2]
```diff
@@ -8831,6 +8899,21 @@
...
 /area/centcom/holding/cafe)
+"nKE" = (
+/obj/structure/table/wood,
+/obj/item/stack/sheet/mineral/gold{
+	amount = 5
+	},
+/obj/item/stack/sheet/mineral/silver{
+	amount = 5
+	},
+/obj/item/stack/sheet/mineral/plasma/thirty{
```
Comment (YakumoChen) [inline _maps/map_files/generic/CentCom_skyrat_z2.dmm]: /obj/item/stack/sheet/mineral/plasma/
not /obj/item/stack/sheet/mineral/plasma/thirty

File: _maps/map_files/generic/CentCom_skyrat_z2.dmm [hunk 3]
```diff
@@ -7824,6 +7848,28 @@
...
+/obj/effect/turf_decal/tile/neutral{
+	dir = 8
+	},
+/obj/effect/turf_decal/trimline/blue/filled/line,
+/obj/machinery/vending/imported/mothic{
+	default_price = 0;
+	extra_price = 0;
+	fair_market_price = 0;
+	extended_inventory = 1
+	},
```
Comment (YakumoChen) [inline _maps/map_files/generic/CentCom_skyrat_z2.dmm:7860]: as above, no need to varedit

File: _maps/map_files/generic/CentCom_skyrat_z2.dmm [hunk 4]
```diff
@@ -8324,6 +8370,28 @@
...
+/obj/effect/turf_decal/tile/neutral{
+	dir = 8
+	},
+/obj/effect/turf_decal/trimline/blue/filled/line,
+/obj/machinery/vending/imported/yangyu{
+	default_price = 0;
+	extra_price = 0;
+	fair_market_price = 0;
+	extended_inventory = 1
+	},
```
Comment (YakumoChen) [inline _maps/map_files/generic/CentCom_skyrat_z2.dmm:8383]: same

File: _maps/map_files/generic/CentCom_skyrat_z2.dmm [hunk 5]
```diff
@@ -10497,6 +10594,29 @@
...
+	dir = 8
+	},
+/obj/effect/turf_decal/trimline/blue/filled/line,
+/obj/machinery/light,
+/obj/machinery/vending/imported/tizirian{
+	default_price = 0;
+	extra_price = 0;
+	fair_market_price = 0;
+	extended_inventory = 1
+	},
```
Comment (YakumoChen) [inline _maps/map_files/generic/CentCom_skyrat_z2.dmm:10603]: same

**Comment by MidoriWroth** [comment]
most of the other machines had those edits on them so i just copied them

**Comment by YakumoChen** [comment]
Remove the chasm please, it doesn't really make sense. You can just put a trash can or something there instead.

**Comment by MidoriWroth** [comment]
chasm??

**Comment by YakumoChen** [comment]
Uhh I thought there was a test merge and an admin had fucked around with the cafe to put in the things you added

There was a chasm as waste disposal for... some reason.

**Review by Paxilmaniac — CHANGES_REQUESTED** [review]
Please don't just leave a stack of uranium sitting in the open on a table, its just going to irradiate anyone who uses the cafe

**Comment by MidoriWroth** [comment]
> Please don't just leave a stack of uranium sitting in the open on a table, its just going to irradiate anyone who uses the cafe

uranium sheets themselves don't cause radiation, it's furniture and other stuff they are built with that causes it

**Comment by YakumoChen** [comment]
uranium isn't dangerous until the stack size reaches over 30 I think

**Comment by MidoriWroth** [comment]
@Paxilmaniac

**Review by Paxilmaniac — APPROVED** [review]

**Pull request merged**